app/models/log.rb

```diff
@@ -0,0 +1,61 @@
+class Log
+  require 'elasticsearch/persistence/model'
+  include Elasticsearch::Persistence::Model
+
+  # TODO: Always capitalize method before saving, etc
+
+  index_name "log_#{Rails.env}"
+
+  attribute :http_method, String
+  attribute :request_id, String
+  attribute :status, String
+  attribute :host, String
+  attribute :path, String, mapping: { analyzer: 'special_characters' }
+  attribute :fwd, String
+  attribute :raw, String
+  attribute :timestamp, DateTime
+
+  def to_s
+    if request_id.blank?
```

Comment:
Why?

Reply:
Well, I guess It's because then the raw log is not in the expected format, but then we need a method name to explain that.